llms/mlx_lm/models/deepseek_v2.py

```diff
@@ -94,7 +93,6 @@ def __init__(
         self.max_position_embeddings = max_position_embeddings
         self.base = base
         self.scaling_factor = scaling_factor
-        self.original_max_position_embeddings = original_max_position_embeddings
```

Comment:
@awni I think the original_max_position_embeddings is still needed, it's used by `yarn_find_correction_range`.

Reply:
Oops, I will add it back.